.github/workflows/nightly_tests.yml

```diff
@@ -180,6 +184,61 @@ jobs:
         pip install slack_sdk tabulate
         python utils/log_reports.py >> $GITHUB_STEP_SUMMARY
 
+  run_big_gpu_torch_tests:
+    name: Torch tests on big GPU (24GB)
```

Comment:
Will we use 24GB or 40? (The decorator has a hard coded value, which is why I'm asking?)

Reply:
40GB. Will update.